text/0000-fabric-private-chaincode-lite.md

```diff
@@ -457,9 +457,17 @@ We recommend reading the chaincode development and deployment sections to know m
 
 In order to focus the development resources on the core components of FPC, the MVP `FPC Lite` excludes certain Fabric features, which will be added in the future.
 
-- Multiple implementations for a single chaincode -- a feature added in Fabric v2.0. 
-***not in current arch, but if you know other versions behave similarly and do not leak... good! but cannot install and use arbitrary package***
-   -- is fundamentally incompatible for private chaincode like FPC where _all_ information flows have to be controlled: for FPC chaincodes to be considered equivalent they must be bit-for-bit identical in order to generate matching identities (i.e. MRENCLAVE).
+- Multiple implementations for a single chaincode.
+This feature is supported in Fabric v2 and gives organizations the freedom to implement and package their own chaincode.
+Ultimately, the channel can use different chaincode implementations as long as these expose the same observable actions (i.e., transaction responses), independently from the actually executed binaries.
```

Comment:
In fabric you do really care only about ledger state (changes), transaction responses is a bit ambiguous (and as far as result is concerned, it is also not really (globally/publically) observable? The small fix would be to replace `transaction responses` by `ledger state changes`.  However, i think it would be better to talk about that what is essential that ledger changes must happen according to the same agreed upon state machine?  E.g., something along the lines of

```suggestion
It allows for different chaincode implementations as long as changes to the ledger state implement the same agreed upon state machine, i.e., the application integrity is ensured.
```